packages/carbon-react/package.json

```diff
@@ -44,10 +44,10 @@
   "dependencies": {
     "@carbon/feature-flags": "^0.6.0",
     "@carbon/icons-react": "^10.38.0",
-    "@carbon/styles": "^0.4.0-rc.0",
+    "@carbon/styles": "^0.4.0",
     "@carbon/telemetry": "0.0.0-alpha.6",
-    "carbon-components": "10.41.0",
```

Comment:
is this right? jumping from `41` to `43`?

Reply:
@jnm2377 yeah, not sure why it was pinned to 41 as it should track latest